site/src/content/docs/getting-started/javascript.mdx

```diff
@@ -258,6 +259,10 @@ const tooltip = new bootstrap.Tooltip(yourTooltipEl, {
 })
 ```
 
+<Callout type="warning">
+**Exercise caution when using these advanced options.** Vulnerabilities caused solely by disabling or modifying content sanitization are not considered within scope for Bootstrap’s security model.
```

Comment:
I would maybe mention again the reference here too. And move this callout just before the sentence "You can add new values to this default `allowList`:"

```suggestion
**Exercise caution when using these advanced options.** Refer to [OWASP’s Cross Site Scripting Prevention Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html) for more information. Vulnerabilities caused solely by disabling or modifying content sanitization are not considered within scope for Bootstrap’s security model.
```